app/forms/remote_search_form.rb

```diff
@@ -0,0 +1,13 @@
+class RemoteSearchForm
+  include ActiveModel::Model
+  include ActiveModel::Validations::Callbacks
+
+  attr_accessor :online_checkbox, :by_phone_checkbox,
```

Comment:
Split these attributes up over separate lines.

Also, can we remove the `_checkbox` suffix. We don't need these hints for the specific types of fields.